views/components/Graphs/BarGraph.jsx

```diff
@@ -29,29 +29,25 @@ const BarGraph = ({ graph }) => {
         y={20}
         labelComponent={<VictoryLabel />}
       />
-      <VictoryAxis
-        label='Site'
-        style={graphStyles.xAxis}
-      />
+      <VictoryAxis label='Site' style={graphStyles.xAxis} />
       <VictoryAxis
         label='Total(%)'
```

Comment:
Since these aren't percentages should we remove the percentage from the label?

```suggestion
        label='Total'
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/115/commits/181384a418ea7628c74eb97994f5992d4f1e3f71